PrincipleStudios.Tools.Ui/src/pages/culture/interviews.mdx

```diff
@@ -0,0 +1,91 @@
+within the software engineering industry. Needless to say, we were curious if
+there was a better way.
+
+We started by looking at the goal of an interview - which, put simply, is "to
+find the ideal candidate for the role." Unfortunately, that definition was too
+broad to be useful. After some iteration, we arrived at four key aspects for
+an ideal software engineer:
+
+- Be aligned with our [company cultural values][1]
+- Have the skills necessary to do the current job today
+- Have the potential to grow to one of the best in the industry
+- Be flexible enough in their work process to accomodate our clients'
+  sometimes-unusual workflows
+
+From there, we decided we wanted each interviewer to be able to focus on just
+one of these aspects, and it'd be easy to find a software engineer or manager to
+interview for each aspect. But, there's more to an interview than just that.
+
+We also want to make sure new hires can work well with the other departments! As
+a result, our two non-technical points (cultural values and flexible process) we
+decided should be performed by non-engineers.
+
+Maybe most importantly, we know that the best software engineers are in high
+demand; as a result, we know they'll have a ton of questions for us about who we
+are, what we do, and what it's like to actually work for us. As a result, our
+interviews are set up to have time as for a two-way interview; we want to make
+sure applicants have plenty of time to make sure that we're a good fit for them.
+Our interviewers talk openly about their own experience at Principle, answering
+questions as honestly as we can while still respecting NDAs of our clients.
+
+Beyond that, each interviewer is encouraged to conduct their interview any way
+they choose to be able to answer their particular question. For the technical
+interviews, we've found an unusual approach that has served us quite well so
+far:
+
+- To see if a candidate has the skills we're looking for currently, we invite
+  them to share code they're passionate about that they've written! We ask them
+  to walk one of us through it, allowing us to ask questions and dive into the
+  parts they're most excited about. Seeing another developer get excited is
+  infectious for our team, and gets us to see the candidate at their best:
+  worries tend to wash away and we all have a great time, knowing exactly how
+  someone can fit into our existing team.
+- To see if a candidate has the potential to grow, we share some of our
+  (non-cient) code and treat it like an onboarding session! Since an interview
+  tends to be one of the most stressful parts of our career, getting someone to
+  learn while stressed gives us a great indicator of how they'll start on any
+  team, but having us admit when our code isn't the best also puts them at ease.
+  After a canned bit of onboarding, we let them look anywhere in that codebase,
+  answering any questions or highlighting some of the parts we're most excited
+  about.
+
```

Comment:
You may want to point out that reading (and maybe finding bugs in) existing code is going to be a much bigger part of the job than writing new code, generally. And, seeing someone explain what existing code does gives a much better sense of their understanding of code and coding practices than having them write something from scratch.